app/controllers/users_controller.rb

```diff
@@ -126,6 +129,29 @@ def buy_barcode
       buy_drink
     end
   end
+  
+  # POST /users/1/print_label
+  # POST /users/1/print_label.json
+  def print_label
+    @user = User.find(params[:id])
+    text = "#{@user.name} #{Date.today.iso8601}"
+    uri = URI('http://labello/print')
```

Comment:
configureable!

Reply:
So, we need a configuration file. But we need this for #5 anyway.